.travis.yml

```diff
@@ -19,7 +19,10 @@ before_install:
   - sudo make base
   - sudo make apt-openfoam-deps apt-simphony-deps apt-lammps-deps apt-mayavi-deps apt-aviz-deps
   - make fix-pip
-  - pip install --allow-external PIL --allow-unverified PIL PIL
+  - sudo pip uninstall -y argparse
```

Comment:
In anycase if these commands are necessary they should be part of the fix-pip or the related target that requires them